util/mongodb.js

```diff
@@ -1,11 +1,15 @@
 import { MongoClient } from 'mongodb';
 
-const { MONGODB_URI, MONGODB_DB } = process.env;
+const { MONGODB_URI, MONGODB_DB, LOCAL_MONGODB, NODE_ENV} = process.env;
 
-if (!MONGODB_URI) {
+if (NODE_ENV !== 'development' && !MONGODB_URI) {
```

Comment:
```suggestion
if (NODE_ENV === 'development' && !MONGODB_URI) {
```
I think we need to raise this only on development. Isn't it?